deployment-templates/compute-engine/client/forseti-instance-client.py

```diff
@@ -24,9 +33,28 @@ def GenerateConfig(context):
         "git clone {src_path}.git".format(
             src_path=context.properties['src-path']))
 
-    FORSETI_VERSION = (
-        "git checkout {forseti_version}".format(
-            forseti_version=context.properties['forseti-version']))
+    patch_search_expression = glob_expr_matching_patches(context.properties['forseti-version'])
+    if patch_search_expression:
+        CHECKOUT_FORSETI_VERSION = (
+            """matches=$(git tag -l {patch_search_expression})
+    matches=(${{matches//;/ }})
+    for match in "${{matches[@]}}"
+    do
+    segments=(${{match//./ }})
+    patch=${{segments[2]}}
+    patch=${{patch: 0: 2}}
+    patch=$(echo $patch | sed 's/[^0-9]*//g')
+    if !((${{#latest_version[@]}})) || ((patch > ${{latest_version[1]}}));
+    then
+      latest_version=($match $patch)
+    fi
+    done
+    git checkout ${{latest_version[0]}}"""
```

Comment:
It took me an hour to get the code in the bash script to have no indentation.  This is the way I found to work.  I am happy to change it if you see it critical but it would take a good amount of playing with and I know we are pressed for time.  Let me know

Reply:
Got it, then okay as it.